index.js

```diff
@@ -0,0 +1,27 @@
+const bodyParser = require('body-parser')
+const jsonParser = bodyParser.json()
+const request = require('request')
+const express = require('express')
+const app = express()
+
+app.use(jsonParser)
+app.use(express.static('public'))
+
+app.post('/url-request', (req, res) => {
+  checkYoutubeId(req.body.youtubeId)
+    .then(response => {
```

Comment:
so this promise is either resolved or rejected, listen for Promise rejections using `.catch`